test/integration/build_admission_test.go

```diff
@@ -23,7 +35,7 @@ func TestPolicyBasedRestrictionOfBuildCreateAndCloneByStrategy(t *testing.T) {
 	builds := map[string]*buildapi.Build{}
 
 	// Create builds to setup test
-	for _, strategy := range buildStrategyTypes() {
+	for _, strategy := range buildStrategyTypesDefault() {
```

Comment:
I'd rather keep trying to create all types and ensure an error is returned for build types not granted by default (applies to all of these tests)